arenadata/Dockerfile.ubuntu

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:22.04 AS base
+
+COPY README.Ubuntu.bash ./
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Why do we need this variable permanently in the image?

Reply:
This allows to not display menus during package install and has no harm for tests. Also environment for the image and container will be closer. I see no reason to not leave it for the image.

Reply:
Yes, but I can connect to a container based on this image and manually run something, and this variable will get in my way!

Reply:
unset DEBIAN_FRONTEND will help you